gsap-simple-tween/js/index.js

```diff
@@ -0,0 +1 @@
+TweenLite.to("div", 2, {scaleY:0.75, scaleX:1.25, y:100, opacity:0.75, ease:Elastic.easeOut});
```

Comment:
may want to comment this out so that students can understand what's going on line by line